torch/hub.py

```diff
@@ -68,6 +68,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 # Copied from tools/shared/module_loader to be included in torch package
 def import_module(name, path):
+    warnings.warn('The use of import_module is deprecated and will be removed in a future release')
```

Comment:
Let's be very explicit here to help users figure out the timeline:


```suggestion
    warnings.warn('The use of torch.hub.import_module is deprecated in v0.11 and will be removed in v0.12')
```